### PR TITLE
Change font to Roboto to support more charsets

### LINKED ISSUE #9801: The font Lato does not support all characters
- Gitea version (or commit ref): 1.12.0+dev-80-g3f1e062d2

## Description
For issue titles the font [*Lato*](https://fonts.google.com/specimen/Lato) is being used. The problem is that this font does not support non-latin characters like "āčēģīķļņšūž" (these are used in Latvian alphabet).

The fonts that are defined in CSS for h1:
```Lato,-apple-system,BlinkMacSystemFont,system-ui,Segoe UI,Roboto,Helvetica,Arial,sans-serif```
Github uses:
```-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji```
...

## Screenshots

image

### PR DESCRIPTION
**Pull request by lafriks**
I can't really reproduce issue, could be my local setup maybe.

Anyway Roboto font supports way more charsets than Lato does

Fixes #9801

Also drops old browser support that we don't support anyway

### Before
image

### After
image

### CODE REVIEW

**Comment by 6543** [comment]
I get
```
GET /vendor/assets/lato-fonts/lato-v16-latin-regular.woff2 404 Not Found
GET /vendor/assets/lato-fonts/lato-v16-latin-700.woff2 404 Not Found
```
in the logs

**Review by sapk — CHANGES_REQUESTED** [review]
This PR need to update librejs and serviceworker cache definition:
https://github.com/go-gitea/gitea/blob/590d56dbb355a499b2f59d9a6ffa4887ea33c96b/public/vendor/librejs.html#L124 
https://github.com/go-gitea/gitea/blob/f00961abe72dbe3922986d22dfa7be2e2261e7be/templates/pwa/serviceworker_js.tmpl#L50

**Comment by lunny** [comment]
@lafriks Fomantic-ui still depend on lato fonts...

**Comment by sapk** [comment]
If I remember well we build fomantic-ui now so we could override some variable like :
https://github.com/Semantic-Org/example-github/blob/master/semantic/src/themes/github/globals/site.variables#L10

**Comment by lunny** [comment]
@sapk yes. so we need #9561

**Comment by silverwind** [comment]
https://github.com/go-gitea/gitea/pull/9561 has landed now

**Comment by silverwind** [comment]
Not sure if worth it but the font could be sourced from https://www.npmjs.com/package/typeface-roboto. It does seem to update sometimes, but probably lags behind the google fonts version https://github.com/KyleAMathews/typefaces/blob/master/packages/roboto/files-hash.json.

**Comment by lafriks** [comment]
@silverwind @sapk @lunny rebased and updated to set roboto font also for fomantic build

File: web_src/less/_base.less [hunk 1]
```diff
@@ -1,85 +1,57 @@
...
-        url('../vendor/assets/lato-fonts/lato-v16-latin-regular.svg#Lato') format('svg');
-}
-
-/* lato-italic - latin */
+        local('Roboto'),
+        local('Roboto-Regular'),
+        /* Chrome 26+, Opera 23+, Firefox 39+ */
+        url('../vendor/assets/roboto-fonts/roboto-v20-latin-ext_cyrillic-ext_latin_greek_vietnamese_cyrillic_greek-ext-regular.woff2') format('woff2'),
+        /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+        url('../vendor/assets/roboto-fonts/roboto-v20-latin-ext_cyrillic-ext_latin_greek_vietnamese_cyrillic_greek-ext-regular.woff') format('woff');
```
Comment (silverwind) [inline web_src/less/_base.less:12]: Would put comments like this, which will help us find those IE compats once its time to remove. No need to comment the woff2.

``` css
        url('../vendor/assets/roboto-fonts/roboto-v20-latin-ext_cyrillic-ext_latin_greek_vietnamese_cyrillic_greek-ext-regular.woff2') format('woff2'),
        /* compat: IE11 */
        url('../vendor/assets/roboto-fonts/roboto-v20-latin-ext_cyrillic-ext_latin_greek_vietnamese_cyrillic_greek-ext-regular.woff') format('woff');
```

**Review by silverwind — APPROVED** [review]

File: web_src/less/_base.less (hunk 1 above)
Comment (lafriks) [reply web_src/less/_base.less:12]: Comments are generated by that font download code, it's same also as for lato

Comment (silverwind) [reply web_src/less/_base.less:12]: Which code? Do we have a font updating script somewhere?

Comment (lafriks) [reply web_src/less/_base.less:12]: This code: https://github.com/go-gitea/gitea/pull/9803/files#diff-3635838a3c33d79a155c7526989c5577R126

Comment (lafriks) [reply web_src/less/_base.less:12]: It's manually copied over

**Review by sapk — APPROVED** [review]

**Comment by zeripath** [comment]
@lafriks I can't update your branch...

**Comment by lafriks** [comment]
Updated, don't know why you can't uodate

**Pull request merged**